dm/dm/master/openapi_test.go

```diff
@@ -129,6 +131,19 @@ func (t *openAPISuite) TestRedirectRequestToLeader(c *check.C) {
 	cancel()
 }
 
+func (t *openAPISuite) TestOpenAPIWillNotStartInDefaultConfig(c *check.C) {
+	// create a new cluster
+	cfg1 := NewConfig()
+	c.Assert(cfg1.Parse([]string{"-config=./dm-master.toml"}), check.IsNil)
+	cfg1.Name = "dm-master-1"
+	cfg1.DataDir = c.MkDir()
+	cfg1.MasterAddr = tempurl.Alloc()[len("http://"):]
+	cfg1.PeerUrls = tempurl.Alloc()
+	cfg1.AdvertisePeerUrls = cfg1.PeerUrls
+	cfg1.InitialCluster = fmt.Sprintf("%s=%s", cfg1.Name, cfg1.AdvertisePeerUrls)
+	c.Assert(NewServer(cfg1).echo, check.IsNil)
```

Comment:
please make sure the old code will fail the test. You didn't `Start` so echo is always nil